### PR TITLE
Activate admin address if `isActivateAdminAddress` is true

### PR DESCRIPTION
**Pull request by riemannulus**
- Activate admin address if `isActivateAdminAddress` is true

### CODE REVIEW

File: .Lib9c.Tools/SubCommend/Create.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public class Create
         public void Genesis(
             [Option('g')] string gameConfigDir,
             [Option('d')] string goldDistributedPath,
-            [Option('a')] int activationKeyCount)
+            [Option('c')] int activationKeyCount)
```
Comment (limebell) [inline .Lib9c.Tools/SubCommend/Create.cs:21]: Is this change necessary? If so, how about separate commit for this change?

Comment (riemannulus) [reply .Lib9c.Tools/SubCommend/Create.cs:21]: Maybe it was my fault. I'll revert it. 😓

**Review by earlbread — APPROVED** [review]

**Review by limebell — APPROVED** [review]
